Assets/scss/components/quesionnaire.scss

```diff
@@ -0,0 +1,250 @@
+.cmp-questionnaire {
+  max-width: toRem(1175);
+  margin: auto;
+
+  @include max-screen(sm) {
+    max-width: toRem(767);
```

Comment:
@chintanpatelrightpoint We don't need this because 767 is already greater than sm

Reply:
Removed the small screen mixin thanks for the feedback